jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildTarTask.java

```diff
@@ -91,7 +91,7 @@ public FileCollection getInputFiles() {
    */
   @OutputFile
   public String getOutputFile() {
-    return getTargetPath().toString();
+    return Preconditions.checkNotNull(jibExtension).getOutputPaths().getTarPath().toString();
```

Comment:
Likewise, will a relative path always work correctly?

Reply:
This one seems to act a bit weird. It looks like for maven, paths are relative to the project root, but for gradle, paths are relative to the directory the command is run from. If I do something like build the tar outside the project directory then try to rebuild inside the project directory, it skips generating the tar the second time. Even stranger is that when I delete the original tar it still refuses to regenerate it. Might be safer to just remove `@OutputFile`.

Reply:
Hmm... I like `@OutputFile` because a tarball can be large and Gradle will avoid generating it. (Now you said it, I wonder if Maven works in the same manner.)

Some options:
- We just say this is a bug to be fixed in Gradle. Sounds like difficult to fix though.
- Don't support relative path. I think this is not a big deal.

Reply:
Actually, I think we can just prepend the project root if this is a relative path. The behavior will be no different from specifying an absolute path.

Reply:
Done.